.changeset/twenty-games-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Sparkle icon added to flight library
```

Comment:
I believe this is [the format we aim for](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#new-icons)
```suggestion
`sparkle` icon added
```